test/integration/verify/controls/azure_container_group.rb

```diff
@@ -0,0 +1,11 @@
+resource_group = input(:resource_group, value: '')
+container_group_name = input(:inspec_container_group_name, value: '')
+
+control 'check container group properties' do
+  describe azure_container_group(resource_group: resource_group, name: container_group_name) do
```

Comment:
After the control statement. We should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
It is not mandatory .. and most integration tests in Azure either way do not follow that ..